build/build.py

```diff
@@ -431,21 +434,30 @@ def main():
           "plugin is still experimental and is not ready for use yet."
       ),
   )
+  parser.add_argument(
+      "--build_gpu_kernel_plugin",
+      choices=["cuda", "rocm"],
+      default="",
+      help=(
+          "Specify 'cuda' or 'rocm' to build the respective kernel plugin."
+          " When this flag is set, jaxlib will not be built."
+      ),
+  )
   add_boolean_argument(
       parser,
-      "build_cuda_kernel_plugin",
+      "build_gpu_pjrt_plugin",
       default=False,
       help_str=(
-          "Are we building the cuda kernel plugin? jaxlib will not be built "
+          "Are we building the cuda/rocm pjrt plugin? jaxlib will not be built "
           "when this flag is True."
       ),
   )
   add_boolean_argument(
       parser,
```

Comment:
is this needed?

Reply:
maybe the latest change is not pushed? `build_rocm_pjrt_plugin` is not needed right?

Reply:
Oops, fixed. Thanks!